---
 configure.ac      |    2 +-
 pidgin/gtkblist.c |    3 +++
 2 files changed, 4 insertions(+), 1 deletion(-)

Index: pidgin-2.7.11/configure.ac
===================================================================
--- pidgin-2.7.11.orig/configure.ac
+++ pidgin-2.7.11/configure.ac
@@ -410,17 +410,17 @@
 fi
 if test X"$x_includes" = X"" || test X"$x_includes" = XNONE; then
     x_incpath_add=
 else
     x_incpath_add="-I$x_includes"
 fi
 
 if test "x$enable_gtkui" = "xyes" ; then
-	PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.10.0], , [
+	PKG_CHECK_MODULES(GTK, [gtk+-2.0 >= 2.10.0 launchpad-integration], , [
 		AC_MSG_RESULT(no)
 		AC_MSG_ERROR([
 
 You must have GTK+ 2.10.0 or newer development headers installed to compile
 Pidgin.  If you want to build only Finch then specify --disable-gtkui when
 running configure.
 ])])
 
Index: pidgin-2.7.11/pidgin/gtkblist.c
===================================================================
--- pidgin-2.7.11.orig/pidgin/gtkblist.c
+++ pidgin-2.7.11/pidgin/gtkblist.c
@@ -64,16 +64,17 @@
 #include "gtkblist-theme-loader.h"
 #include "gtkutils.h"
 #include "pidgin/minidialog.h"
 #include "pidgin/pidgintooltip.h"
 
 #include <gdk/gdkkeysyms.h>
 #include <gtk/gtk.h>
 #include <gdk/gdk.h>
+#include <launchpad-integration.h> 
 
 typedef struct
 {
 	PurpleAccount *account;
 	GtkWidget *window;
 	GtkBox *vbox;
 	GtkWidget *account_menu;
 	GtkSizeGroup *sg;
@@ -5789,16 +5790,18 @@
 	gtk_window_add_accel_group(GTK_WINDOW (gtkblist->window), accel_group);
 	g_object_unref(accel_group);
 	gtkblist->ift = gtk_item_factory_new(GTK_TYPE_MENU_BAR, "<PurpleMain>", accel_group);
 	gtk_item_factory_set_translate_func(gtkblist->ift,
 										(GtkTranslateFunc)item_factory_translate_func,
 										NULL, NULL);
 	gtk_item_factory_create_items(gtkblist->ift, sizeof(blist_menu) / sizeof(*blist_menu),
 								  blist_menu, NULL);
+	launchpad_integration_add_items (gtk_item_factory_get_widget(gtkblist->ift, N_("/Help")), -1, TRUE, TRUE);
+
 	pidgin_load_accels();
 	g_signal_connect(G_OBJECT(accel_group), "accel-changed", G_CALLBACK(pidgin_save_accels_cb), NULL);
 
 	menu = gtk_item_factory_get_widget(gtkblist->ift, "<PurpleMain>");
 	gtkblist->menutray = pidgin_menu_tray_new();
 	gtk_menu_shell_append(GTK_MENU_SHELL(menu), gtkblist->menutray);
 	gtk_widget_show(gtkblist->menutray);
 	gtk_widget_show(menu);
