=== modified file 'pidgin/gtkdocklet.c'
--- old/pidgin/gtkdocklet.c	2009-03-09 23:42:38 +0000
+++ new/pidgin/gtkdocklet.c	2009-03-19 04:03:25 +0000
@@ -841,7 +841,7 @@
 
 	purple_prefs_add_none(PIDGIN_PREFS_ROOT "/docklet");
 	purple_prefs_add_bool(PIDGIN_PREFS_ROOT "/docklet/blink", FALSE);
-	purple_prefs_add_string(PIDGIN_PREFS_ROOT "/docklet/show", "always");
+	purple_prefs_add_string(PIDGIN_PREFS_ROOT "/docklet/show", "never");
 	purple_prefs_connect_callback(docklet_handle, PIDGIN_PREFS_ROOT "/docklet/show",
 				    docklet_show_pref_changed_cb, NULL);
 

