From: Steve Magoun
Subject: irc: periodic '/who' polling causes connection drops
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/pidgin/+bug/856631

Index: pidgin-2.10.0/libpurple/protocols/irc/msgs.c
===================================================================
--- pidgin-2.10.0.orig/libpurple/protocols/irc/msgs.c	2011-09-21 13:44:32.532391000 -0400
+++ pidgin-2.10.0/libpurple/protocols/irc/msgs.c	2011-09-21 13:45:43.282494768 -0400
@@ -110,8 +110,10 @@
 	irc_blist_timeout(irc);
 	if (!irc->timer)
 		irc->timer = purple_timeout_add_seconds(45, (GSourceFunc)irc_blist_timeout, (gpointer)irc);
+    /*
     if (!irc->who_channel_timer)
         irc->who_channel_timer = purple_timeout_add_seconds(300, (GSourceFunc)irc_who_channel_timeout, (gpointer)irc);
+    */
 }
 
 void irc_msg_default(struct irc_conn *irc, const char *name, const char *from, char **args)
